pkg/catalog/catalog_test.go

```diff
@@ -0,0 +1,121 @@
+package catalog
+
+import (
+	"reflect"
+	"testing"
+
+	imagev1 "github.com/openshift/api/image/v1"
+	"github.com/redhat-developer/odo/pkg/occlient"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	ktesting "k8s.io/client-go/testing"
+)
+
+// Function taken from occlient_test.go
+// fakeImageStream gets imagestream for the reactor
+func fakeImageStream(imageName string, namespace string, tags []string) *imagev1.ImageStream {
```

Comment:
We should move these helper functions to a different file under `occlient` package and just call them from there. WDYT? We could take up that exercise post PR merge too. 

Cc: @kadel

Reply:
Yeah, I was trying to move them to `fakeclient.go` or making them public within `occlient_test.go` (which didn't work).

To be honest, I had to heavily modify these functions to work specifically for `catalog list` (since I'm passing in `tags []string` and modifying it. I would be more keen towards leaving them in the `catalog_test.go` file.

Reply:
Those helper function should be moved to something like `testingutil` pkg and merged into just one function, so we can share it between different tests. 
Ideally in this PR :grimacing: If not then we should at least open new Issue so we don't forget to clean this.

Reply:
@kadel No worries, I'll move them to a testingutil pkg :)